vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors_moe.py

```diff
@@ -951,6 +951,9 @@ def select_gemm_impl(
 
             return experts
 
+        # aiter path
+        from vllm.model_executor.layers.fused_moe import AiterExperts
```

Comment:
Should this be moved to the `elif self.rocm_aiter_moe_enabled` block?  Otherwise we might be trying to import it in cases where aiter is not available?

If `AiterExperts` is always safe to import then you can ignore this comment.

Reply:
done